connector/src/main/java/org/geysermc/connector/network/translators/java/entity/spawn/JavaPlayerListEntryTranslator.java

```diff
@@ -26,8 +25,13 @@ public void translate(ServerPlayerListEntryPacket packet, GeyserSession session)
             PlayerListPacket.Entry entry1 = new PlayerListPacket.Entry(entry.getProfile().getId());
 
             if (packet.getAction() == PlayerListEntryAction.ADD_PLAYER) {
+                if (session.getPlayerEntity().getUuid().equals(entry.getProfile().getId())) continue;
                 long geyserId = session.getEntityCache().getNextEntityId().incrementAndGet();
 
+                SetLocalPlayerAsInitializedPacket initPacket = new SetLocalPlayerAsInitializedPacket();
+                initPacket.setRuntimeEntityId(geyserId);
+                session.getUpstream().sendPacket(initPacket);
```

Comment:
This packet is only server-bound. You need to wait for the client to send this before adding new players to the list.

Reply:
Ah shit i made that mistake too

Reply:
Will fix that